data/souhaibsfaxi.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "YOUR NAME",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/souhaibsfaxi.png",
+  "tags": ["javascript", "typescript"],
+  "socials": [
+    
+    { "platform": "github", "url": "https://github.com/souhaibsfaxi" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
```

Your GitHub username must be specified here.
```suggestion
      "url": "https://github.com/souhaibsfaxi",
```